scripts/get-dep-lib.sh

```diff
@@ -102,27 +105,27 @@ if [ "$PLATFORM" == "singleAndroid" ] ; then
 fi
 # no curl and old libs (never version breaks)
 # check whether to use curl or wget
-if [ "`which curl`" == "" ] ; then
+if [ "$(which curl)" == "" ] ; then
 	CURL="wget "
 else
 	CURL="curl -O "
 fi
-BUILD_COMMON="libzip libgit2 googlemaps"
+COMMON_PACKAGES=(libzip libgit2 googlemap)
 case ${PLATFORM} in
 	scripts)
-		BUILD="${BUILD_COMMON} hidapi libcurl libusb openssl libssh2"
+		PACKAGES=("${BUILD_COMMON[@]}" hidapi libcurl libusb openssl libssh2)
 		;;
 	ios)
-		BUILD="${BUILD_COMMON} libxslt"
+		PACKAGES=("${BUILD_COMMON[@]}" libxslt)
 		;;
 	android)
-		BUILD="${BUILD_COMMON} libxslt sqlite libxml2 openssl libftdi libusb"
+		PACKAGES=("${BUILD_COMMON[@]}" libxslt sqlite libxml2 openssl libftdi libusb)
```

Comment:
Typo. Rather forgot it here. Nothing uses the android keyword, so it wasn't found by the ci.

Reply:
it is correct, the build.sh's does not use it. But I use it frequently to prefetch everything for a build, because then the build can run without internet while I travel.